### PR TITLE
Move plot widget to bottom

### PR DESCRIPTION
**Pull request by ahuang11**
Builds on top of https://github.com/holoviz/hvplot/pull/1198 for more screen space for the plot and partially addresses https://github.com/holoviz/hvplot/issues/1190

image

### CODE REVIEW

**Review by maximlt — COMMENTED** [review]
I don't exactly know why but it doesn't fit very well in a notebook output cell. Do you interact with the explorer always as an app? I'd guess the explorer is more used in a notebook context.

image

When served as an app the plot occupies the whole space and the widget box looks a little alone centered down there.

image

What alternative do we have? We could maybe dynamically create a new tab populated with the widget box? Or add it below the groupby multiselect?

**Comment by ahuang11** [comment]
A floating pane or built into Bokeh's toolbar if ever possible

**Comment by ahuang11** [comment]
Oh I suppose we could do `widget_layout = pn.Row` instead of `pn.Column`.

**Comment by ahuang11** [comment]
image

**Comment by ahuang11** [comment]
Another thing we need to fix is the default `legend_loc`.

File: hvplot/ui.py [hunk 1]
```diff
@@ -528,9 +531,10 @@ def __init__(self, df, **params):
                 # Using .layout on the HoloViews pane to display the widgets
                 # https://github.com/holoviz/panel/issues/5628#issuecomment-1763443895
                 pn.Tabs(('Plot', self._hv_pane.layout), ('Code', self._code_pane)),
-                sizing_mode='stretch_width',
+                sizing_mode='stretch_both'
             ),
-            sizing_mode='stretch_both'
+            sizing_mode='stretch_width',
+            height=600
```
Comment (ahuang11) [inline hvplot/ui.py:537]: `min_height` unfortunately did not work for me.

**Comment by maximlt** [comment]
> image

The widgets are not well vertically aligned, I don't think we can merge with this issue, let's see if I can fix it.

**Comment by maximlt** [comment]
Came up with a workaround:
image

**Review by maximlt — APPROVED** [review]
The explorer will look much better with this change when there are multiple widgets!

**Comment by ahuang11** [comment]
Shall we merge?

**Comment by maximlt** [comment]
Yes, I was waiting for the tests to pass.

**Pull request merged**